src/bmeg/edge.py

```diff
@@ -199,3 +199,48 @@ class DrugResponseIn(Edge):
     DrugResponse -> Biosample
     """
     pass
+
+
+@enforce_types
+@dataclass(frozen=True)
+class HasSupportingReference(Edge):
+    """
+    G2PAssociation -> Publication
+    """
+    pass
+
+
+@enforce_types
+@dataclass(frozen=True)
+class GeneFeatureFor(Edge):
```

Comment:
To me this class name (and AlleleFeatureFor, PhenotypeOf, etc) make me expect the edge to point in the opposite direction.

Maybe HasGeneFeature?